data/en/census_household.json

```diff
@@ -193,7 +197,26 @@
+                                    "guidance": {
+                                        "show_guidance": "Show naming guidance",
+                                        "hide_guidance": "Hide naming guidance",
+                                        "content": [
+                                            { "description": "Enter the current first, middle and last names of all the people who usually live here." },
+                                            { "description": "If you have not yet named your newborn, please enter ‘Baby’ for their first name and then enter their last name (surname)." },
+                                            { "description": "Please also include household members who have requested a personal form." },
+                                            { "description": "Please also include the names of people staying with you temporarily who don’t have a usual address in the UK. This is so that we make sure we count everyone." },
+                                            {
+                                                "title": "Include",
+                                                "list": [
+                                                    "People who usually live outside the UK who are staying in the UK for <strong>three months or more</strong>",
+                                                    "People who work away from home within the UK  if this is their permanent or family home",
+                                                    "Members of the Armed Forces if this is their permanent or family home",
+                                                    "People who are temporarily outside the UK for <strong>less than 12 months</strong>",
+                                                    "People staying temporarily who usually live in the UK but do not have another UK address, for example relatives, friends"
+                                                ]
+                                            }
+                                        ]
+                                    },
```

Comment:
So this was all on the one line with html inline and as it stands this is the only one with this structure, it looks like the better way to have it, but this is where the high level review would come in.